tables/create_archive_manager.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE archive_manager
+    (   first_name VARCHAR(32),
+        last_name VARCHAR(32), 
+        handle VARCHAR(16), 
+        email  VARCHAR(255), 
+        access_level VARCHAR(32), 
+        date_joined  DATE,
```

Comment:
See if we can do this,
We need a default value here, which is the date when we insert the values in this table.